src/%IPM/Main.cls

```diff
@@ -155,75 +188,106 @@ repo -n LocalFiles -delete
+<modifier name="list" description="Lists all repositories (possibly subject to a type filter: e.g. -filesystem, -remote, -local)" />
 <modifier name="list-modules" aliases="modules,listmodules" description="List modules available in the specified repository (-n\[ame]), or in all configured repositories if no repository was specified." />
-
+<modifier name="info" description="Shows configuration details for the current namespace as a remote repository." />
+<modifier name="copy-from" value="true" description="Copies repository configuration from a different (specified) namespace" />
 <!-- General/shared modifiers -->
 <modifier name="name" aliases="n" value="true" description="Namespace-unique name for the module" />
 <modifier name="delete" description="Deletes the current namespace's reference to the named repository" />
 <modifier name="publish" value="true" valueList="0,1" description="When configuring a remote repository, specifies that publishing of packages to the repository is allowed. When configuring the current namespace with -enable, specifies that publishing is also enabled. "/>
-
 <!-- General properties -->
 <modifier name="snapshots" aliases="s" value="true" valueList="0,1" description="For any repository, specifies that it should be used to look for snapshot builds (i.e., those with a semantic version ending in '+snapshot', indicating a 'latest' build of a particular version)." />
 <modifier name="prereleases" aliases="pre" value="true" valueList="0,1" description="For any repository, specifies that it should be used to look for prerelease software" />
 <modifier name="read-only" aliases="ro" value="true" valueList="0,1" description="For any repository, specifies that modules installed from it are ALWAYS installed in non-developer mode, effectively making them read only" />
-
+<!-- Repository types  -->
 <modifier name="type" value="true" aliases="t" description="Subclass of %IPM.Repo.Definition to create/modify/delete or implementation of %IPM.Repo.IPackageService or %IPM.Repo.IPublishService to enable/disable" />
 
 </command>
 
 <command name="load" dataPrefix="D">
-<description>Loads a module from the specified directory or archive into the current namespace. Dependencies are also loaded automatically, provided that they can be found in repositories configured with the 'repo' command.</description>
+<summary>
+Loads a module from the specified directory into the current namespace.
+</summary>
+<description>
+Loads a module from the specified directory into the current namespace.
+Dependencies are also loaded automatically, provided that they can be found in
+repositories configured with the 'repo' command.
+</description>
+
+<!-- Examples -->
 <example description="Loads the module described in C:\module\root\path\module.xml">
 load C:\module\root\path\
-load C:\module\root\path\module-0.0.1.tgz
```

Comment:
why, this was deleted?

Reply:
Thank you Dmitry and good catch! I miss delete it in a merge, now it's added back now!